### PR TITLE
Hook up invites to subscription model

### PR DESCRIPTION
**Pull request by mbrandonw**
Now that we have live subscription data I can derive invites stuff from that info.

This also makes it possible to remove someone from a team.

While talking about this today I realized we also need to show this info to teammate and allow them to leave the team. I may add that to this PR or do it separately.

### CODE REVIEW

File: Sources/PointFree/Database.swift [hunk 1]
```diff
@@ -120,9 +122,9 @@ private func add(userId: Database.User.Id, toSubscriptionId subscriptionId: Data
   return execute(
     """
     UPDATE "users"
-    SET "subscription_id" = $1,
+    SET "subscription_id" = $2,
         "updated_at" = NOW()
-    WHERE "users"."id" = $2
+    WHERE "users"."id" = $1
```
Comment (mbrandonw) [inline Sources/PointFree/Database.swift]: whoops!

File: Sources/PointFree/Account/Account.swift [hunk 2]
```diff
@@ -233,16 +233,23 @@ private let inviteRowView = View<Database.TeamInvite> { invite in
...
-      ]),
-    gridColumn(sizes: [.mobile: 10], [
-      div([`class`([Class.padding([.mobile: [.leftRight: 1]])])], [
-        p(["You have 10 open spots on your team. Invite a team member below:"]),
+private let subscriptionInviteMoreRowView = View<(Stripe.Subscription?, [Database.TeamInvite], [Database.User])> { subscription, invites, teammates -> [Node] in
+
+  guard let subscription = subscription else { return [] }
+  guard subscription.quantity > 1 else { return [] }
+  let invitesRemaining = subscription.quantity - invites.count - teammates.count
+  guard invitesRemaining > 0 else { return [] }
```
Comment (stephencelis) [inline Sources/PointFree/Account/Account.swift]: Nice way of short-circuiting this view!

File: Sources/PointFree/Database.swift (hunk 1 above)
Comment (stephencelis) [reply Sources/PointFree/Database.swift]: Lol! Maybe flip the args instead?

**Comment by mbrandonw** [comment]
will merge once I get the green!

**Pull request merged**